### DISCUSSION TITLE
Respec(finalbuild) condition

### DISCUSSION THREAD

**Discussion by mirakhany** [isid0re/SoloLeveling#288] [Q&A]
Hi. my sorceress lvl is 90ish but still use leveling build skills (blizzard, fireball, gspike, meteor)
Where can I find respec condition for finalbuild?
I want to see and compare condition :D

**Comment by isid0re** [comment]
Respec condition is in globals.js setup respecTwo. 

There is a small issue with check.haveitem() for the tals pieces. I suspect it's a spelling issue on the names of the items. And have it on my todo list.

**Reply by isid0re** [reply]
The fix has been pushed so you can update your files

**Reply by mirakhany** [reply]
Yes. my sorceress started respec and changed to final Meterob build.
Thank you so much!

**Comment by randy-cruz** [comment]
I'm having trouble understanding how respecTwo works. For example: Poison, Bone and Summon Necromancers all have the same (blank) condition. I don't see where they're given a level to trigger a respec. I can see levels defined in case you're running Classic mode, however.

What am I missing?

**Reply by isid0re** [reply]
They all share the same condition in LOD they need enigma. If no enigma no respec. Since lod respecs are based on gear.

**Reply by randy-cruz** [reply]
Where can I see that? I've been digging through the code and trying to trace it out.

**Reply by Neural90** [reply]
Conditions are stored in functions/globals.js

**Comment by King-Noob** [comment]
hey bud. at first respect for the work u done. i was proud when i fuggured out to have more than i char in one game :-) i downloaded all a week ago and startet 3 sorc to compare the advantages
cold
blizzballer
blova
all are act 3 hell and still on startersetup (blizzard, fireball, gspike, meteor). am i wright that all have to be lvl 75 for finalk respec to continue the act? they are doing pit und chest all the time....and does the blizzballer sorc need 4 of 5 tal set to change skill

**Reply by isid0re** [reply]
Needs 4 pieces.you can look at globals.js for the respecTwo trigger
